py/sentry_streams/flink/flink_adapter.py

```diff
@@ -53,3 +57,16 @@ def sink(self, step: Step, stream: Any) -> Any:
         )
 
         return stream.sink_to(sink)
+
+    def map(self, step: Step, stream: Any) -> Any:
```

Comment:
This currently assumes a specific format from users for how they define the path to their Map function

Will add checks for this

Reply:
Not sure I get the issue. Are you talking about validating that the path to the function is actually a valid one ?
Isn't the check what you are doing at line 70 ?

Reply:
Outdated comment, sorry